apps/frontend/src/pages/FormPage.tsx

```diff
@@ -26,21 +65,47 @@ const FormPage: React.FC = () => {
     >
       {(form) => (
         <Form>
-          <GrantFormPage />
-
-          <Spacer height={16} />
+          {step === 1 && (
+            <FormBranching
+              currentStep={grantFormCurrentStep}
+              form={GrantFormPage}
+              confirmationQuestion={'Are you a parent or legal guardian?'}
+              endMessage={'Please find a parent or legal guardian to complete the form.'}
+              onConfirm={handleGrantForm}
+            />
+          )}
 
-          <MedicalFormPage />
+          {step === 2 && (
+            <FormBranching
+              currentStep={medicalFormCurrentStep}
+              form={MedicalFormPage}
+              confirmationQuestion={'Are you a medical professional?'}
+              endMessage={'Please find a medical professional to complete the form.'}
+              onConfirm={handleMedicalProfessional}
+            />
+          )}
 
-          <Center>
-            <Button mt={4} colorScheme="teal" onClick={form.submitForm}>
-              Submit
-            </Button>
+          <Center mt={4}>
```

Comment:
Maybe consider making these buttons a separate component. Not necessary though

Reply:
Updated in most recent commit